tests/models/test_licensing.py

```diff
@@ -76,15 +77,35 @@ def test_content_type_media_type(self):
             self.audiobook_drm_scheme.content_type_media_type)
 
     def test_default_fulfillable(self):
+        # Try some well-known media type/DRM combinations known to be
+        # fulfillable by the default client.
+        for media, drm in (
+            (MediaTypes.EPUB_MEDIA_TYPE, DeliveryMechanism.ADOBE_DRM),
+            (MediaTypes.EPUB_MEDIA_TYPE, DeliveryMechanism.NO_DRM),
+            (MediaTypes.EPUB_MEDIA_TYPE, DeliveryMechanism.BEARER_TOKEN),
+            (MediaTypes.PDF_MEDIA_TYPE, DeliveryMechanism.NO_DRM),
+            (MediaTypes.PDF_MEDIA_TYPE, DeliveryMechanism.BEARER_TOKEN),
+            (None, DeliveryMechanism.FINDAWAY_DRM),
+            (MediaTypes.AUDIOBOOK_MANIFEST_MEDIA_TYPE, DeliveryMechanism.NO_DRM),
+            (MediaTypes.AUDIOBOOK_MANIFEST_MEDIA_TYPE, DeliveryMechanism.BEARER_TOKEN),
+        ):
+            # All of these DeliveryMechanisms were created when the
+            # database was initialized.
+            mechanism, is_new = DeliveryMechanism.lookup(self._db, media, drm)
+            eq_(False, is_new)
+            eq_(True, mechanism.default_client_can_fulfill)
+
+        # It's possible to create new DeliveryMechanisms at runtime,
+        # but their .default_client_can_fulfill will be False.
         mechanism, is_new = DeliveryMechanism.lookup(
-            self._db, Representation.EPUB_MEDIA_TYPE,
+            self._db, MediaTypes.EPUB_MEDIA_TYPE,
```

Comment:
Beyond this point I'm just changing `Representation` to `MediaTypes`, which is where these constants are actually stored.